runtime/src/iree/io/formats/safetensors/testdata/generate_safetensors_files.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# https://huggingface.co/docs/safetensors/index
+#
+# To regenerate:
+#  $ pip install torch safetensors
```

Comment:
could also put this in a requirements.txt file, possibly pinned to a known-working version if unstable

(fine to ignore, I don't think we'll have enough people updating these regularly enough to need more than this comment)

Reply:
yeah I don't want to pull in big/shady deps - my hope is these never change unless the file format changes so only one person will ever need to install them/rerun this

Reply:
Ah, I mean a requirements.txt _in this directory_, like https://github.com/openxla/iree/blob/main/docs/website/requirements.txt. That lets people install local requirements from any directory without needing to read through the files to see what is needed, and also lets us pin versions or specify minimums. Individual files would not affect the regular build of the python runtime or general requirements for using IREE.